pinot-timeseries/pinot-timeseries-spi/src/main/java/org/apache/pinot/tsdb/spi/plan/LeafTimeSeriesPlanNode.java

```diff
@@ -72,7 +71,9 @@ public String getKlass() {
 
   @Override
   public String getExplainName() {
-    return EXPLAIN_NAME;
```

Comment:
Can we make the changes to trigger the dump of physical and logical plan from client for timeseries? As of now we do not have a way to dump similar to explain in SQL query.

Reply:
Remember we just added a new explain mode that is quite more useful that the normal one we have in V2: https://github.com/apache/pinot/pull/13733

Reply:
Yeah I think we should wait for the multistage-engine integration before explain plan support, so we have a cohesive way to compute them. The new explain mode is quite cool and we should support similar semantics for the time-series engine too. I'll be raising a proposal this month.